pkg/util/os/atexit/atexit.go

```diff
@@ -29,6 +30,13 @@ func Exit(code int) {
 	os.Exit(code)
 }
 
+// ExitWithMessage exit and print the error message.
```

Comment:
Right, an additional Exit method. Since Go does not have overloading we need to provide a new name or use a variadic method.